### PR TITLE
Adding parameter to make transform publishing optional

### PR DESCRIPTION
**Pull request by ayrton04**
@servos @efernandez @afakihcpr

### CODE REVIEW

**Comment by afakihcpr** [comment]
:+1:

File: src/ros_filter.cpp [hunk 1]
```diff
@@ -460,66 +461,69 @@ namespace RobotLocalization
...
+              mapOdomTrans_.mult(worldBaseLinkTrans, odomBaseLinkTrans_);
+
+              mapOdomTransMsg_.transform = tf2::toMsg(mapOdomTrans_);
+              mapOdomTransMsg_.header.stamp = filteredPosition.header.stamp + tfTimeOffset_;
+              mapOdomTransMsg_.header.frame_id = mapFrameId_;
+              mapOdomTransMsg_.child_frame_id = odomFrameId_;
+
+              worldTransformBroadcaster_.sendTransform(mapOdomTransMsg_);
+            }
+            catch(tf2::TransformException ex)
```
Comment (efernandez) [inline src/ros_filter.cpp]: `catch(const tf2::TransformException& ex)`

Comment (ayrton04) [inline src/ros_filter.cpp]: Fixed, thanks!

**Comment by efernandez** [comment]
:+1:

**Comment by servos** [comment]
LGTM

**Pull request merged**